src/components/BuildConfigCreateEditPage/BuildConfigCreateEditPage.tsx

```diff
@@ -387,6 +406,113 @@ export const BuildConfigCreateEditPage = ({ isEditPage = false }: IBuildConfigCr
         </Form>
       </ContentBox>
 
+      <ContentBox padding marginBottom>
+        <Form
+          onSubmit={(e) => {
+            e.preventDefault();
+          }}
+        >
+          <FormGroup
+            isRequired
+            label={`${buildConfigEntityAttributes.scmRepository.title} URL`}
```

Comment:
Please share as much as possible content like this with SCM Repositories create page - for example all title and tooltip attributes. 

The same as valid for other common parts like validation, "blue box" displaying information whether repository is already synced or not (let's export it as separated component). Please create follow-up jiras to apply required changes on SCM Repositories create page.

The goal is to this section and SCM Repositories create page synced as much as possible.